libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -196,6 +202,15 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     // @Path: ../libraries/AP_Param_Helper/AP_Param_Helper.cpp
     AP_SUBGROUPINFO(param_helper, "", 12, AP_BoardConfig, AP_Param_Helper),
 #endif
+
+#if AP_FEATURE_SAFETY_BUTTON
+    // @Param: SAFETYOPTION
+    // @DisplayName: Options for safety button behavior
+    // @Description: This controls the activation of the safety button. It allows you to control if the safety button can be used for safety enable and/or disable, and whether the button is only active when disarmed
+    // @Bitmask: 0:ActiveForSafetyEnable,1:ActiveForSafetyDisable,2:ActiveWhenDisarmed
+    // @User: Standard
```

Comment:
`Standard`?

Reply:
standard so it is grouped with BRD_SAFETYENABLE, which I think it should be in the MP UI